pkgs/tools/filesystems/bcachefs-tools/default.nix

```diff
@@ -59,10 +73,9 @@ stdenv.mkDerivation {
     rm tests/test_fuse.py
   '';
 
-  # this symlink is needed for mount -t bcachefs to work
+  # this wrap is needed for mount -t bcachefs to work
   postFixup = ''
-    ln -s $out/bin/mount.bcachefs.sh $out/bin/mount.bcachefs
-    wrapProgram $out/bin/mount.bcachefs.sh \
+    wrapProgram $out/bin/mount.bcachefs \
```

Comment:
I moved this back to draft since the tests revealed I need a proper fix for this.